src/plugins/intel_cpu/src/nodes/concat.cpp

```diff
@@ -342,6 +342,41 @@ void Concat::prepareParams() {
             hasOuterLoop = true;
         }
     }
+
+    execSpecialCase = nullptr;
+    if (outputShape.size() == 1 && outputStrides[0] == 1 && outputShape[0] <= 64 && elemSize == 4) {
+        // output is small 1d vector (which is typical in shape inference subgraph),
+        // in this case, inputs are also small 1d vector and single thread naive impl is faster
+        bool useSpecial = true;
```

Comment:
Maybe it worth to rename `Special` to `use1DCase` and `exec1DCase`.  Better explains applicability of the impl.
Same comment for other files.

Reply:
Sure, done.